ci-operator/step-registry/baremetalds/e2e/ovn/OWNERS

```diff
@@ -1,4 +1,4 @@
 approvers:
 - sdn-approvers
```

Comment:
Looks like `sdn-approvers` is renamed in `OWNERS_ALIASES`, but still used here ?

Reply:
fixed